### PR TITLE
MM5-6499 added translations to delete and reassign modal

### PR DESCRIPTION
**Pull request by YevheniiKravchenko**

### CODE REVIEW

**Comment by Poiar** [comment]
ms-teams_afKzioztyo

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 1]
```diff
@@ -11842,6 +11842,86 @@ resource configservice_label administration_tools_system_asset_categories_delete
...
+  ]
+}
+
+resource configservice_label administration_tools_system_asset_categories_reassign_modal_body {
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_CATEGORIES_REASSIGN_MODAL_BODY'
+  group = 'administration-tools - system - asset-categories'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Do you want to reassign these assets?'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:11883]:
```suggestion
      default_translation = 'Move the assets to this category:'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

**Comment by bbedigi** [comment]
I suggest a combination
Screenshot 2024-11-11 at 14 08 55

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 2]
```diff
@@ -11858,6 +11858,88 @@ resource configservice_label administration_tools_system_asset_categories_delete
...
+  ]
+}
+
+resource configservice_label administration_tools_system_asset_categories_reassign_modal_body {
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_CATEGORIES_REASSIGN_MODAL_BODY'
+  group = 'administration-tools - system - asset-categories'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Move the assets to this category:'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:11901]:
```suggestion
      default_translation = 'Assign the assets to this category:'
```

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 3]
```diff
@@ -11858,6 +11858,88 @@ resource configservice_label administration_tools_system_asset_categories_delete
...
+  ]
+}
+
+resource configservice_label administration_tools_system_asset_categories_reassign_modal_confirm {
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_CATEGORIES_REASSIGN_MODAL_CONFIRM'
+  group = 'administration-tools - system - asset-categories'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Reassign assets and delete category'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:11933]:
```suggestion
      default_translation = 'Reassign assets and delete category'
```

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 4]
```diff
@@ -11858,6 +11858,88 @@ resource configservice_label administration_tools_system_asset_categories_delete
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Gentildel aktiver og slet kategori'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:11937]:
```suggestion
      default_translation = 'Flyt assetsne og slet kategori'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 5]
```diff
@@ -11858,6 +11858,88 @@ resource configservice_label administration_tools_system_asset_categories_delete
...
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'One or more assets are assigned to this category or its subcategories.
+Choose another category for them to continue.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Et eller flere assets er tildelt denne kategori eller dens underkategorier.
+Vælg en ny kategori for assetsne, for at fortsætte.'
```
Comment (bbedigi) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:11889]:
```suggestion
      default_translation = 'Et eller flere assets er tildelt kategorien \'{{name}}\' eller dens underkategorier.'
```

**Review by bbedigi — APPROVED** [review]

**Review by Poiar — APPROVED** [review]

**Pull request merged**